scenarios/test_codegen/src/handlers/EventHandlers.ts

```
@@ -0,0 +1,762 @@
+expectType<
+  TypeEqual<
+    typeof getBalance,
+    Effect<{ address: string; blockNumber?: bigint | undefined }, BigDecimal>
+  >
+>(true);
+
+const zeroAddress = "0x0000000000000000000000000000000000000000";
+
+Gravatar.CustomSelection.handler(async ({ event, context }) => {
+  console.log("AAAA", event.transaction.hash);
+  if (0) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)